python/tvm/driver/build_module.py

```diff
@@ -136,7 +98,7 @@ def lower(
 
     Parameters
     ----------
-    input : Union[schedule.Schedule, PrimFunc, IRModule]
+    inputs : Union[schedule.Schedule, PrimFunc, IRModule]
```

Comment:
Why is the name `inputs` when it is a singular schedule, primfunc, or irmodule?

Reply:
The word input is a python built in function so you can't name variables `input.` Initially the name of the parameter to the function was `inputs` and the name of the parameter in the documentation was `input`. I just changed the documentation to match the function signature. I don't like that they are different. I agree that `inputs` is not an ideal name though and I'm open to suggestions for other names

Reply:
Maybe use `inp` instead? I can't really think of a good name either.

Reply:
Hi @electriclilies, I guess even if `input` is a builtin function in python, it can still be a parameter name? For example this code should work:
```
def f(input=2):
    print(input)
```
Please correct me if I am wrong. :)

Reply:
The problem is actually that the linter doesn't like it -- I guess it just flags everything called input for some reason.
https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-8110/14/pipeline

Reply:
Thanks for the detail! :) It's a pylint warning so you can disable it using `# pylint: disable=redefined-builtin`. One example is here: https://github.com/apache/tvm/blob/main/python/tvm/topi/image/dilation2d.py#L27.

Reply:
oh, thanks! Well I just changed it to `inp`, so I think I'll leave it that way.

Reply:
Sounds good!